This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: brodieG <brodieG@users.noreply.github.com>
Date: Tue, 21 Jun 2022 01:17:43 +0000
Subject: [PATCH 1/2] add lbz2 to pkgconfig

mabrand: moved to Libs.private and added bcrypt too

diff --git a/minizip.pc.cmakein b/minizip.pc.cmakein
index 1111111..2222222 100644
--- a/minizip.pc.cmakein
+++ b/minizip.pc.cmakein
@@ -8,7 +8,7 @@ Name: @MINIZIP_TARGET@
 Description: Zip manipulation library
 Version: @VERSION@
 
-Requires.private: @PC_PRIVATE_DEPS@
+Requires.private: @PC_PRIVATE_DEPS@ openssl
 Libs: -L${libdir} -L${sharedlibdir} -l@MINIZIP_TARGET@
-Libs.private:@PC_PRIVATE_LIBS@
+Libs.private:@PC_PRIVATE_LIBS@ -lbcrypt -lbz2
 Cflags: -I${includedir}

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Tue, 8 Apr 2025 13:38:14 +0200
Subject: [PATCH 2/2] avoid "unused parameter" error


diff --git a/minizip.c b/minizip.c
index 1111111..2222222 100644
--- a/minizip.c
+++ b/minizip.c
@@ -303,7 +303,7 @@ int32_t minizip_add(const char *path, const char *password, minizip_opt *options
 
 /***************************************************************************/
 
-int32_t minizip_extract_entry_cb(void *handle, void *userdata, mz_zip_file *file_info, const char *path) {
+int32_t minizip_extract_entry_cb(void */* handle */, void *userdata, mz_zip_file *file_info, const char *path) {
     minizip_opt *options = (minizip_opt *)userdata;
     char *utf8_string = NULL;
 
